core/controlplane/config/templates/cloud-config-controller

```diff
@@ -820,6 +832,37 @@ write_files:
           - nonResourceURLs: ["*"]
             verbs: ["*"]
 
+  - path: /srv/kubernetes/rbac/cluster-roles/bootstrapped-node.yaml
+    content: |
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1alpha1
+        metadata:
+            name: bootstrapped-node
+        rules:
+          - apiGroups: ["*"]
+            resources:
+            - nodes
+            verbs:
+            - patch
+            - update
+          - apiGroups: ["*"]
+            resources:
+            - replicasets
+            verbs:
+            - get
+          - apiGroups: ["*"]
+            resources:
+            - replicationcontrollers
+            verbs:
+            - get
+          - apiGroups: ["*"]
+            resources:
+            - pods/eviction
+            verbs:
+            - create
+          - nonResourceURLs: ["*"]
+            verbs: ["*"]
+
```

Comment:
Unfortunately, yes.

The certificates generated via the certificate signing workflow are bound to the `system:nodes` group, which do not have those permissions, which are needed by the node drainer and the node labeler (the one that adds AWS metadata as nodes labels and annotations).

Without this, node draining does not work, and the AWS metadata is not added to nodes bootstrapped via this CSR.